pkg/overlay/server.go

```diff
@@ -159,10 +171,15 @@ func (o *Server) populate(ctx context.Context, startID storj.NodeID, maxNodes, r
 		}
 
 		rest := v.GetRestrictions()
-		if rest.GetFreeBandwidth() < restrictedBandwidth || rest.GetFreeDisk() < restrictedSpace {
-			continue
-		}
-		if contains(excluded, v.Id) {
+		rep := v.GetReputation()
+
+		if rest.GetFreeBandwidth() < restrictions.GetFreeBandwidth() ||
```

Comment:
I think I get that `rest` is the node's restrictions but I'm not totally getting where the `restrictions.GetFreeBandwidth()` is coming from for comparison